data/data/bootstrap/files/usr/local/bin/installer-masters-gather.sh

```diff
@@ -9,6 +9,16 @@ fi
 ARTIFACTS="/tmp/artifacts-${GATHER_ID}"
 mkdir -p "${ARTIFACTS}"
 
+echo "Gathering master systemd summary ..."
```

Comment:
will this be useful for bootstrap gather too?

Reply:
> will this be useful for bootstrap gather too?

Sure; added with 133ea844d -> 8830652bb.